### PR TITLE
I key toggles infinite nextTurn

### PR DESCRIPTION
**Pull request by big-caballito**
just adds convenience

### CODE REVIEW

File: game_handler.cpp [hunk 1]
```diff
@@ -286,6 +286,16 @@ void GameManager::reset(const GameField& field) {
   update();
 }
 
+void GameManager::step() {
```
Comment (kirpichik) [inline game_handler.cpp]: step -> infinitySteps

File: game_handler.cpp [hunk 2]
```diff
@@ -286,6 +286,16 @@ void GameManager::reset(const GameField& field) {
...
+  size_t counter = 0;
```
Comment (kirpichik) [inline game_handler.cpp:292]: Remove unused counter

File: game_handler.cpp [hunk 3]
```diff
@@ -286,6 +286,16 @@ void GameManager::reset(const GameField& field) {
...
+  for (; true; ++counter) {
```
Comment (kirpichik) [inline game_handler.cpp]: Replace with while(true)

**Review by kirpichik — CHANGES_REQUESTED** [review]
Hi @big-caballito!

Thanks for your improvement, but I would like to ask for a few changes. I left some of them in the comments to the code, and here's another one:
It would be nice to add smth like that:
`getViewHandler().updateCommandLine("Making steps... Press I for interrupt.");`
before starting infinity steps and clean cmd prompt after interruption.

File: game_handler.cpp (hunk 2 above)
Comment (kirpichik) [reply game_handler.cpp:292]: Or not delete but display the counter when the user interrupts

**Comment by big-caballito** [comment]
Sorry it took me so long :shrug:

**Comment by kirpichik** [comment]
Do not worry and thank you for your contribution!

**Pull request merged**